client/my-sites/ads/form-earnings.jsx

```diff
@@ -81,198 +71,229 @@ module.exports = React.createClass( {
+		const earnings = this.state.earnings && this.state.earnings.total_earnings ? Number( this.state.earnings.total_earnings ) : 0,
 			owed = this.state.earnings && this.state.earnings.total_amount_owed ? Number( this.state.earnings.total_amount_owed ) : 0,
-			paid = this.state.earnings && this.state.earnings.total_earnings && this.state.earnings.total_amount_owed ?
-				( this.state.earnings.total_earnings - this.state.earnings.total_amount_owed ) : 0;
+			paid = this.state.earnings && this.state.earnings.total_earnings && this.state.earnings.total_amount_owed
+				? this.state.earnings.total_earnings - this.state.earnings.total_amount_owed
+				: 0;
 
 		return (
-			<ul className="earnings_breakdown__list" >
-				<li className="earnings_breakdown__item">
-					<span className="earnings_breakdown__label">{ this.translate( 'Total earnings', { context: 'Sum of earnings' } ) }</span>
-					<span className="earnings_breakdown__value">${ this.numberFormat( earnings, 2 ) }</span>
+			<ul className="ads__earnings-breakdown-list">
+				<li className="ads__earnings-breakdown-item">
+					<span className="ads__earnings-breakdown-label">
+						{ translate( 'Total earnings', { context: 'Sum of earnings' } ) }
+					</span>
+					<span className="ads__earnings-breakdown-value">${ numberFormat( earnings, 2 ) }</span>
 				</li>
-				<li className="earnings_breakdown__item">
-					<span className="earnings_breakdown__label">{ this.translate( 'Total paid', { context: 'Sum of earnings that have been distributed' } ) }</span>
-					<span className="earnings_breakdown__value">${ this.numberFormat( paid, 2 ) }</span>
+				<li className="ads__earnings-breakdown-item">
+					<span className="ads__earnings-breakdown-label">
+						{ translate( 'Total paid', { context: 'Sum of earnings that have been distributed' } ) }
+					</span>
+					<span className="ads__earnings-breakdown-value">${ numberFormat( paid, 2 ) }</span>
 				</li>
-				<li className="earnings_breakdown__item">
-					<span className="earnings_breakdown__label">{ this.translate( 'Outstanding amount', { context: 'Sum earnings left unpaid' } ) }</span>
-					<span className="earnings_breakdown__value">${ this.numberFormat( owed, 2 ) }</span>
+				<li className="ads__earnings-breakdown-item">
+					<span className="ads__earnings-breakdown-label">
+						{ translate( 'Outstanding amount', { context: 'Sum earnings left unpaid' } ) }
+					</span>
+					<span className="ads__earnings-breakdown-value">${ numberFormat( owed, 2 ) }</span>
 				</li>
 			</ul>
 		);
-	},
+	}
 
-	earningsTable: function( earnings, header_text, type ) {
-		var period,
-			rows = [],
+	earningsTable( earnings, header_text, type ) {
+		const { numberFormat, translate } = this.props;
+		const rows = [],
 			infoIcon = this.getInfoToggle( type ) ? 'info' : 'info-outline',
 			classes = classNames( 'earnings_history', {
 				'is-showing-info': this.getInfoToggle( type )
 			} );
+		let period;
```

Comment:
We don't need `period` in this scope, I believe we should be able to declare it as a `const` in the `for` loop:

```js
for ( const period in earnings ) {
```